data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/metrics/Metrics.java

```diff
@@ -116,6 +116,11 @@ public static MetricsOptions getOptions(final BaseEnv metricsConfigs) {
     final var registry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
     return new MicrometerMetricsOptions()
       .setEnabled(true)
+      .addDisabledMetricsCategory(MetricsDomain.HTTP_CLIENT)
+      .addDisabledMetricsCategory(MetricsDomain.HTTP_SERVER)
+      .addDisabledMetricsCategory(MetricsDomain.VERTICLES)
+      .addDisabledMetricsCategory(MetricsDomain.NET_CLIENT)
+      .addDisabledMetricsCategory(MetricsDomain.NET_SERVER)
       .addDisabledMetricsCategory(MetricsDomain.EVENT_BUS)
       .addDisabledMetricsCategory(MetricsDomain.DATAGRAM_SOCKET)
       // NAMED_POOL allocates a lot, so disable it.
```

Comment:
+1 

we could factor those into its own method for a more compact execution - looks like the more we add, the more verbose this gets.

Just a comment :sweat_smile: